src/types.d.ts

```diff
@@ -19,6 +19,7 @@ export interface IRawMetric {
 }
 
 export interface IEnhancedMetric extends IRawMetric {
+  signal_version: string | null;
```

Comment:
What's the reason for going with `string | null` for optionals? I know that you cannot serialize `undefined` in JSON, but I would think that this can be handled whenever the conversion happens.

Reply:
(I did see that the other strings follow the same approach)